test/test.go

```diff
@@ -256,6 +256,15 @@ func (e *TestEnv) Finish() {
 	}
 }
 
+func LowercaseHeaders(h http.Header) http.Header {
+	lowercaseHeaders := make(http.Header, len(h))
+	for h, v := range h {
```

Comment:
can we rename the method parameter `h` to `headers` ? This way it won't shadow the `h` inside the for loop.

Reply:
good call